dojo/tools/twistlock/parser.py

```diff
@@ -58,6 +58,7 @@ def get_item(vulnerability, test):
     # create the finding object
     finding = Finding(
         title=vulnerability['id'] + ": " + vulnerability['packageName'] + " - " + vulnerability['packageVersion'],
+        cve=vulnerability['id'],
```

Comment:
argh, sorry.. actually thought about it, and slipped my mind. Thanks @dr3dd589 !

Reply:
np :smiley: